x/provider/types/msgs.go

```diff
@@ -30,6 +31,9 @@ func (msg MsgCreateProvider) ValidateBasic() error {
 	if msg.Owner.Empty() {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, "MsgCreate: Invalid Provider Address")
 	}
+	if len(msg.Attributes) == 0 {
```

Comment:
It's okay for providers to not have any attributes.

Reply:
Is that something we might want to consider change/enforce a spec down the road? If a Provider has no attributes, then it has nothing to match/filter on; total wildcard. Or am I missing some use case?

Reply:
As a tenant, I can declare the attributes that I want providers that bid on my orders to have.